schema/workflow.yaml

```diff
@@ -693,7 +695,8 @@ $defs:
         oauth2:
           type: object
           oneOf:
-            - properties:
+            - title: OAuth2Info 
```

Comment:
To be more consistent, I'd name ot something like `OAuth2AuthenticationInfo`, even though I'm aware that auth would therefore be repeated. Reason is that `OAUTH2` is the scheme's name.